### PR TITLE
666 - Ticked lists on iOS12

### PR DESCRIPTION
**Pull request by jbdtky**

### CODE REVIEW

File: src/sass/components/ion-content.scss [hunk 1]
```diff
@@ -14,4 +14,8 @@ ion-content {
   &.padded-bottom-cta-with-summary {
     bottom: 134px;
   }
+
+  &.scroll-content {
+    margin-top: -2px;
```
Comment (jbdtky) [inline src/sass/components/ion-content.scss]: By default it is -1px, I adjusted with -2px to match perfectly with the orange bar from at top bar.

Comment (brendoncoin) [reply src/sass/components/ion-content.scss]: I don't understand this change. The header bar is 44px high, and the content is absolutely positioned at 44px from the top.  Seems to me  like a margin of 0 would be about right (perhaps they made it -1px to enforce scrolling?) What's the need you had that caused you to make this change?

File: www/css/main.css [hunk 2]
```diff
@@ -13688,6 +13688,13 @@ action-sheet .bp-action-sheet__backdrop {
   #tab-buy-bitcoin small a:link {
     text-decoration: underline; }
 
+#tab-buy-bitcoin .terms {
```
Comment (brendoncoin) [inline www/css/main.css]: Should this file be included?

**Comment by jchau207** [comment]
TestBuild

**Pull request merged**